app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class='card mx-auto my-1' style='max-width: 440px;'>
+  <div class='card-body'>
+    <h5 class='card-title'> <%= link_to "Post #{i + 1}", user_post_path(@user, post) %></h5>
+    <p class='card-title'><%= post.title%></p>
+    <p class='card-text'><%= post.text%></p>
+    <span>Comments <%= post.comments_counter%></span>
+    <span>Likes <%= post.likes_counter%></span>
+  </div>
+  <ul class='list-group list-group-flush'>
+    <% post.comments.each do |comment| %>
+      <%= render partial: 'comment', locals: { comment: comment } %>
+    <% end %>
+  </ul>
+</div>
+<div>
```

Comment:
Your design
![image](https://github.com/PabloEGonz/blog-app/assets/78251927/b7c0ff97-c1ab-47dc-ac25-532fbc5e5515)

![image](https://github.com/PabloEGonz/blog-app/assets/78251927/396d974c-3605-4e64-b31d-bc623c65bf44)

According to the wireframe, please use the `post.recent_comments` to render the 5 most recent comments in the `app/views/posts/index.html.erb` page 🛠.

Only post show page displays the post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) 🤓